Examples_Nuget/QuickstartUdf/Program.cs

```diff
@@ -0,0 +1,203 @@
+﻿using System;
+using System.Collections.Generic;
+using TileDB.Cloud;
+using TileDB.Cloud.Rest.Model;
+
+namespace QuickstartUdf
+{
+    class Program
+    {
+        static UDFInfo UdfGetInfo(string uri)
+        {
+            return TileDB.Cloud.RestUtil.Udf.GetInfo(uri);
+        }
+
+        // Run generic UDF using JSON to pass args
+        // + (Python) UDF declaration for this example: def args_udf(arg1, arg2="default")
+        static void UdfGenericExec(string udfUri, string chargedOrg=null, string args=null)
+        {
+            Console.WriteLine($"Running generic UDF {udfUri}...");
+            DebugOutput(udfUri, chargedOrg, args);
+
+            var result = TileDB.Cloud.RestUtil.Udf.ExecGeneric(udfUri, args, chargedOrg: chargedOrg);
+            // Output result to console
+            Console.Write("Result from generic UDF: ");
+            result.CopyTo(Console.OpenStandardOutput());
+            Console.WriteLine("\n");
+        }
+
+        static void UdfGenericExecAsync(string udfUri)
+        {
+            Console.WriteLine($"Running generic UDF {udfUri} asynchronously...");
+            // Call UDF asynchronously; Does not block execution on call to SubmitGenericUDFAsync()
+            var resultTask = TileDB.Cloud.RestUtil.Udf.ExecGenericAsync(udfUri, chargedOrg: "TileDB-Inc");
+            // Here we are free to do some work while waiting on resultTask...
+            Console.WriteLine("Started async UDF task...");
+
+            Console.Write($"Async UDF task completed with output: ");
+            // Accessing the `Result` property of `taskResponse` will block until async task completes
+            resultTask.Result.Result.CopyTo(Console.OpenStandardOutput());
+            Console.WriteLine("\n");
+        }
+
+        // Run Array UDF using array as input; `data` arg in the UDF below will contain array slice data
+        // + (Python) UDF declaration for this example: def array_udf(data, attr, scalar)
+        static void UdfArrayExec(string udfUri, string arrayUri, string attribute, int scalar, string chargedOrg=null)
+        {
+            Console.WriteLine($"Running {udfUri} against {arrayUri}...");
+            var args = RestUtil.SerializeArguments(
```

Comment:
I think ideally we should encapsulate this serialization step inside the middle layer rather than requiring users to do it manually. Can be left for follow-up, if feasible - please create an issue.

Reply:
Yeah, I think we could probably use [params](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/params) for this, but I would need to do some refactoring. I created a story to track this.

[SC-21629](https://app.shortcut.com/tiledb-inc/story/21629)